src/components/app/ListOfPublishedProfiles.css

```diff
@@ -38,11 +38,9 @@
   display: flex;
   justify-content: space-between;
 
-  /* The padding compensates for the negative margin. The negative margin is
-   * there so that the left border (implemented as a box-shadow below) go
-   * outside the line. */
-  padding: 2px 0;
-  margin: -2px 0;
+  /* Ensure that the left "focus-within" border (implemented as a box-shadow below) goes
+   * outside button's focus border. */
+  padding: 1px 0;
```

Comment:
This look simpler than my previous solution :+1: :) I couldn't see a behaviour difference, am I supposed to see one?